test/simulator/timestamp/src/session_simulator.cpp

```diff
@@ -28,4 +28,22 @@
 
 #include "session_simulator.h"
 
+#include <iostream>
+
 session_simulator::session_simulator() {}
+
+int
+session_simulator::begin_transaction()
+{
+    /*
+     * Check no other transactions are running. There should be a 1:1 relationship between session
+     * and transaction.
```

Comment:
How about?
`Cannot begin a new transaction as there is an existing transaction that is currently running. `